test/stream.cc

```diff
@@ -259,6 +271,7 @@ TEST(StreamTest, InterruptLoop) {
         | Loop<int>()
               .context(Lazy<std::atomic<bool>>(false))
               .interruptible()
+              .raises()
```

Comment:
Why `raises()` instead of `raises<std::runtime_error>()` which is what it looks like it raises on line 297?